### ISSUE TITLE
Display SNAP as monthly

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-us#528]

**Comment by fedderw** [comment]
I was thinking about this for Medicaid as well - eligibility is usually based on monthly income. Is that true for SNAP?

**Comment by MaxGhenis** [comment]
Yes, all benefits I'm aware of consider monthly income, but we also show taxes which are yearly, and higher-income users will be more interested in annual numbers. We probably want a toggle to show everything in monthly or yearly terms on the household page.
